docs/sdkx-cocos2dx/getting-started.mdx

```diff
@@ -0,0 +1,308 @@
+    implementation(name: 'Helpshift', ext:'aar')
+}
+```
+
+3.  Link Helpshift files in your project as source depending on your build system.
+
+<Tabs>
+
+<TabItem value="Using CMake" label="Using CMake">
+
+    # Add Helpshift files to CMakeLists.txt, under the Android dependency
+    if(ANDROID)
+      list(APPEND GAME_SOURCE
+           <other dependency files>
+           Classes/HelpshiftCocos2dx.cpp
+           Classes/HelpshiftUtil.cpp
+           Classes/HelpshiftLog.cpp
+           )
+      list(APPEND GAME_HEADER
+          <other dependency files>
+          Classes/HelpshiftCocos2dx.h
+          Classes/HelpshiftUtil.h
+          Classes/HelpshiftLog.h
+          Classes/HelpshiftEvent.h
+          )
+    endif()
+
+</TabItem>
+<TabItem value="Using ndk-build" label="Using ndk-build">
+
+    # Add Helpshift files to makefile (Android.mk), under the LOCAL_SRC_FILES rule
+    LOCAL_SRC_FILES := hellocpp/main.cpp \
+                      ../../../Classes/HelpshiftCocos2dx.cpp \
+                      ../../../Classes/HelpshiftUtil.cpp \
+                      ../../../Classes/HelpshiftLog.cpp \
+
+</TabItem>
+
+</Tabs>
+
+</TabItem>
+<TabItem value="ios" label="iOS">
+
+1. Unzip the package, add the contents of the `ios/Classes` folder into your Xcode project.
+2. From `ios/sdk` folder drag & drop `Helpshift.xcframework` into the Frameworks, Libraries and Embedded content in the General section of your Xcode project.
+3. If your game does not contain any swift code, set `Always Embed Swift Standard Libaries` under `Build Settings` to YES.
+4. In the Build Settings of your Xcode project, add `-fmodules` and `-fcxx-modules` flags under the `Other C Flags` section in both Debug and Release. Be careful while adding these flags in Debug and Release, because if you override the default flags you might face build errors. The `Other C Flags` section would look this after adding mentioned flags:
+   &nbsp;&nbsp; <Image src="/static/books/cocos/integration-c-flag.png"  width="full" />
+
+#### If you are cocos2dx 4.x add extra flag:
+
+In the Build Settings of your Xcode project, add `-fmodules` and `-fcxx-modules` flags under the `Other C++ Flags` section in both Debug and Release. Be careful while adding these flags in Debug and Release, because if you override the default flags you might face build errors.
+
+<Admonition type="info" title="Note">
+
+#### Add `NSPhotoLibraryUsageDescription` key in your application's info.plist file
+
+If your game does not use this permission, you would need to add this key as well as description for the same. Not adding this key-description pair might cause game rejection.
+
+**NSPhotoLibraryUsageDescription Description text:** "We need to access photos library to allow users manually pick images meant to be sent as attachment for help and support reasons."
+
+</Admonition>
+
+<Admonition type="info" title="Note">
+
+The following instructions apply only to **versions below SDK X 10.2.0**.
```

Comment:
do we need this section in Cocos plugin ?
It will be at or above 10.2.0 right ?

Reply:
Yes, will remove this